src/playercards/PlayerCardSpawner.ttslua

```diff
@@ -1,3 +1,9 @@
+
+-- Amount to shift for the next card (zShift) or next row of cards (xShift)
+-- Note that the table rotation is weird, and the X axis is vertical while the
+-- Z axis is horizontal
+local zShift = -2.29998
```

Comment:
Does this need that many decimals?

Reply:
Nope.  Was trying to match the Memory Bag layout exactly, but certainly overkill.